charts/argo-rollouts/templates/argo-rollouts-deployment.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
 spec:
   selector:
     matchLabels:
+      app.kubernetes.io/component: {{ .Values.controller.component }}
```

Comment:
This one here is a breaking change and requires the `--force`  parameter when using `helm upgrade`. I am unsure how we solve this.
Options:
- bump major chart version
- place a upgrade note inside the README
- ...?

Any ideas? @oliverbaehler , @davidkarlsen ?

Reply:
Bump Major and add a Upgrading Section to the README.md (like https://github.com/argoproj/argo-helm/tree/master/charts/argo-cd#upgrading)

Reply:
@mkilchhofer , the major version is bumped with added upgrading note. Please take a look. Thanks.